src/client/components/KudoForm/KudoForm.tsx

```diff
@@ -0,0 +1,147 @@
+import React, { RefObject } from 'react';
+import SelectSearch from 'react-select-search';
+import { CARD_TYPE } from '../../../common/constants';
+import { CardIcon } from '../cardIcon/CardIcon';
+import data from '../../assets/data';
+import './KudoForm.css';
+
+const enum FORM_ERROR {
+  name = 'name',
+  message = 'message'
+}
+
+export interface ISelectResponse {
+  name: string;
+  value: string;
+  index: number;
+}
+
+const CARD_TYPES = Object.values(CARD_TYPE);
+
+const PEOPLE = [...data];
+PEOPLE.map((folk: any) => {
```

Comment:
do not remap to xx_yyy, it is not necessary, value can have spaces if it is a string.

Reply:
do it responsive, fix validation for empty string text